CONTRIBUTING.md

```diff
@@ -1,3 +1,13 @@
+### Project License:  Eclipse Public License v1.0
+
+- You will only Submit Contributions where You have authored 100% of the content.
+- You will only Submit Contributions to which You have the necessary rights. This means that if You are employed You have received the necessary permissions from Your employer to make the Contributions.
```

Comment:
Interesting read, thanks. I guess it is being stated unspecific on purpose.

How about you and David for example. Do you have some kind of "permission" from Google to work on JUnit?